doc/usage.md

```diff
@@ -158,6 +157,20 @@ made to the copy-on-write image, and not to the backing file. Only the
 modified data will use actual disk space.
 A copy-on-write image can be created using `cp -c` or [clonefile(2)](http://www.manpagez.com/man/2/clonefile/).
 
+#### Cloud-init
+
+The `--device virtio-blk` option could also be used to supply an initial configuration to cloud-init through an disk image.
+
+The ISO image file must be labelled CIDATA and it must contain the user-data and meta-data files. 
```

Comment:
Have you tried to use `--device virtioblk,deviceId=CIDATA` for this? (not asking for any changes or for more testing, just curious)

Reply:
yeah I was wondering if `-volid cidata` can be forced on vfkit command line using `deviceId`